src/extensions/coblocks-settings/index.js

```diff
@@ -1,19 +1,13 @@
 /*global coblocksSettings*/
-
-/**
- * Styles
- */
-import './styles/style.scss';
```

Comment:
Styles should not be imported directly.

Reply:
@jrtashjian can you double check? I think there was a specific reason for that.

Reply:
If that is the case, this was duplicated already here:
https://github.com/godaddy-wordpress/coblocks/pull/2189/files#diff-0baec429b7317cde7bf9145f886b8402d07d4143756b8288e98c7958ad295f78R26

Reply:
yeah, it doesn't matter anymore with the new wp-scripts setup. Any file named `index.scss` will be merged into the editor stylesheet, and `style.scss` will be merged into the general stylesheet.